client/src/components/review_create/BookDashboard.jsx

```
@@ -0,0 +1,17 @@
+import BookDetailContent from '../content/BookDetailContent';
+
+export default function BookDashboard({ bookDetail }) {
+  const { imageUrl } = bookDetail;
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.bookCoverWrapper}>
+        <img src={imageUrl} alt="bookImageUrl" />
+      </div>
+      <div className={styles.bookContentWrapper}>
+        <BookDetailContent bookDetail={bookDetail}></BookDetailContent>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)